lvmd/command/lvm.go

```diff
@@ -687,13 +565,17 @@ func (l *LogicalVolume) Resize(newSize uint64) error {
 	if err := CallLVM("lvresize", "-L", fmt.Sprintf("%vb", newSize), l.fullname); err != nil {
 		return err
 	}
-	l.size = newSize
+
+	l.vg.Update()
```

Comment:
if err := l.vg.Update(); err != nil {
		return err
	}

Reply:
@tasleson Please reflect my comments and fix the failure of the build test. Then I'll approve this PR.

Reply:
All changes and lint correction in latest PR.  I also rebased it and squashed it a bit.  Thanks